backend/LeetWars.Builder/RunnerDefaults/DefaultRunnerImageNames.cs

```diff
@@ -6,6 +6,21 @@ public static class DefaultRunnerImageNames
 
         public const string CSharpImage = "csharpcoderunner";
 
+        public const string CSharpTestImage = "csharptestrunner1";
+
+        public const string CSharpBuildImage = "csharpcodebuilder";
+
         public const string JSImage = "jscoderunner";
     }
+
+    public static class CSharpFileNaming
+    {
+        public const string SolutionFileName = "Solution.cs";
+
+        public const string SolutionTestFileName = "Tests.cs";
+
+        public const string TestResultsFileName = "testresults.trx";
+
+        public const string BuildErrorResultsFileName = "errbuildoutput.txt";
+    }
```

Comment:
where do you use it?

Reply:
It was not mine, but from a parallel branch but because I do not use it I deleted it if something comes it would just add again to the project

Reply:
I guess you could use it in solutionRunnerService